dask/array/utils.py

```diff
@@ -28,6 +28,22 @@ def normalize_to_array(x):
         return x
 
 
+def meta_nonempty(x):
+    """Normalize an array to appropriate non-empty meta object
+
+    Parameters
+    ----------
+    x: array-like
+        An object that looks sufficiently like a Numpy array
+    Returns
+    -------
+    array-like with one element of the correct dtype and ndims
+    """
+    if not is_arraylike(x):
+        return x
+    return np.ones((1,) * x.ndim, dtype=x.dtype)
```

Comment:
Perhaps this should use [np.ones_like[(https://numpy.org/doc/stable/reference/generated/numpy.ones_like.html) instead.  cc @pentschev

Reply:
If this is indeed necessary, we would probably want https://github.com/dask/dask/blob/8f9fc1635170cda07a4bb8c4bd93e96da151d938/dask/array/utils.py#L340-L349 here, but I'm wondering why can't we just use `meta_from_array` instead.